docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+
+The verification system has three possible layers of detectors.
+You layer them together so that objects detected in the images your verification's system camera captures point to countdown, alarmed, or disarmed states.
+With our recommended combination of detectors, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.
+
+For example:
+
+{{<video webm_src="/tutorials/verification-system/demo.webm" mp4_src="/tutorials/verification-system/demo.mp4" alt="Bijan interacting with the verification system" poster="/tutorials/verification-system/trigger_1.png">}}
+
+## Requirements
+
+- A [camera](/components/camera/)
+- A computer running `viam-server` that the camera can connect to, like a [single-board computer (SBC)](/components/board/)
+
+Before configuring your camera, you must [create a robot](/fleet/machines/#add-a-new-robot).
+
+1. On the locations page of the [Viam app](https://app.viam.com), add a new robot by providing a name in the **New Robot** field and clicking **Add robot**.
+2. Click on that robot's name to go to its page.
+
+## Configure a camera
+
+Navigate to the **Config** tab of your machine's page on the [Viam app](https://app.viam.com).
+Configure the camera you want to use for your security system.
+
+We configured ours as a webcam named `"my-webcam"`.
+Follow [these instructions](/components/camera/webcam/) to do so.
+
+Connect your camera to your machine running `viam-server`, and hang it up somewhere where it can see people in your home:
+
+{{<imgproc src="/tutorials/verification-system/camera.jpeg" resize="500x" declaredimensions=true alt="Camera hanging up in office.">}}
+
+Next, configure the person detector, or, the "coarser" layer of the security system that verifies that there's a person moving.
+
+## Configure an `mlmodel` person detector
+
+Create a detector to detect when people are in your verification system camera's field of vision:
+
+1. [Capture images from your camera](/data/capture/#configure-data-capture-for-individual-components) with Viam's data management service, then [create a dataset](/data/dataset/#create-a-dataset-and-add-data).
+2. Label your data with [bounding boxes](/data/dataset/#bounding-boxes) labeled as `Person` where a person is present to create an object detection model for people. You only want this model to be able to distinguish between what is and isn't a person, so you can conduct this training step with anyone.
+3. [Train a model on your dataset](/ml/train-model/). Select **Model Type** as **Object Detection**.
+4. [Deploy the model](/ml/deploy/) to your machine so you can use it.
+
+Now that you've trained and deployed the model itself, [configure an `mlmodel` detector model](/ml/vision/detection/#configure-an-mlmodel-detector) with the name of the ML model service you deployed your model as.
```

Comment:
Here I think you need to add more detailed steps as well.  You put multiple steps in one sentence that people have to figure out for themselves, instead of really guiding the reader

Reply:
Done